packages/bpk-component-image/stories.js

```diff
@@ -39,14 +39,35 @@ storiesOf('bpk-component-image', module)
       src={image}
     />
   ))
-    .add('Full Width', () => (
-      <BpkImage
-        altText="image"
-        width={612}
-        height={408}
-        src={image}
-      />
-    ))
+  .add('Full Width', () => (
+    <BpkImage
+      altText="image"
+      width={612}
+      height={408}
+      src={image}
+    />
+  ))
+  .add('Using SrcSet', () => (
+    <BpkImage
+      altText="image"
+      width={612}
+      height={408}
+      srcSet={`${image}?resize=280px:280px&quality=100 280w,
```

Comment:
AFAIK you still need to supply a `src` attribute when using `srcset`...

Reply:
FYI it works without, but probably best practice to!

Reply:
Yup, as a fallback for older browsers 👍